modules/thermal_hydraulics/test/tests/problems/brayton_cycle/tests

```diff
@@ -57,4 +57,32 @@
       detail = 'for a long duration.'
     []
   []
+
+  [recuperated]
+    requirement = 'The system shall be able to model an open recuperated Brayton cycle'
+    [light]
+      type = CSVDiff
+      input = 'recuperated_brayton_cycle.i'
+      csvdiff = 'recuperated_brayton_cycle_light.csv'
+      cli_args = 'Executioner/num_steps=5 Outputs/csv/file_base=recuperated_brayton_cycle_light'
+      abs_zero = 1e-10
+      rel_err = 1e-5
+      max_parallel = 1
```

Comment:
@joshuahansel Any reason we should keep that restriction?  I see the tests above still have it. With ` mesh_mode = 'replicated'` it should be pass.

Reply:
I agree, I'm guessing `max_parallel = 1` just wasn't removed because I put the test in while David was still working on it (he started from an earlier branch). @freytp try replacing `max_parallel = 1` with `mesh_mode = 'replicated'` for these tests.

Reply:
Do you want me to change them to all the tests, or just the new ones that I made?

Reply:
@joshuahansel so when I changed the open and closed cycle tests to `mesh_mode = replicated` it breaks the Modules Recover test

Reply:
The differences appear to be round-off. The tolerances need to be relaxed. Set `abs_zero = 1e-7` for all of the tests.

Reply:
@lindsayad Just a heads up that you'll get a conflict in the Brayton file test specs because of my above suggestion.

Reply:
Oh no. I don't want to have to mess with this stuff again. Please make sure that any additional Brayton tests are solved very tightly or that auxiliary information is not output for exodiff/csvdiff comparison. The auxiliary information is *extremely* sensitive to how tightly you solve these problems. I think that in order to get Brayton auxiliary results that are robust against residual object execution ordering I had to turn off relative tolerancing (e.g. `nl_rel_tol = 1e-50`) and set `nl_abs_tol = 1e-11`

Reply:
@freytp See Alex's changes to the Brayton cycle tests in https://github.com/idaholab/moose/pull/21198. He found that the iteration adaptive DT was not robust in testing, so he switched to a constant dt. You need to do the same.

@lindsayad Were you seeing relative errors of like 1e-5 for quantities that were around 1e-10? That's what Thomas was seeing for example.